### ISSUE TITLE
Cannot find module './cli.js'

### ISSUE THREAD

**Issue by jackyoung022** [twentyhq/twenty#3228]
## Bug Description

When I was following the docs to deploy twenty by Docker, there was this Error in my machine:
![](https://img.jackyra.com/image/2024/01/4e8a1f65bfbd002914134c408f7af8f5.png)
I tried in MacOS and CentOS, same issue.

**Comment by johnyang33** [comment]
I'm also getting this same issue on MacOS
<img width="1728" alt="Screenshot 2024-01-04 at 3 37 34 AM" src="https://github.com/twentyhq/twenty/assets/126351732/ad1f8efd-0774-4bc7-98c7-61f6a9c6194f">

**Comment by Jeong-Min-Cho** [comment]
![image](https://github.com/twentyhq/twenty/assets/9837449/e1759f00-5795-4c8f-92bf-a75985710871)

I have encountered the same issue on WSL (Ubuntu 20.04.5 LTS), but **not** on Docker.
It was the **local setup**.

From me, it was fixed by resetting the workspace to the original by `git reset --hard` then execute `yarn nx database:init twenty-server` again.

It was perfectly working fine yesterday with the same setup, so I guess some left-over caches can cause a problem?

I will come back to this issue to see how to reproduce the issue and solve it properly.
